pkgs/stdenv/generic/setup.sh

```diff
@@ -205,11 +205,16 @@ isScript() {
 }
 
 # printf unfortunately will print a trailing newline regardless
-printLines() {
+printWords() {
     [[ "$#" -gt 0 ]] || return 0
     printf '%s\n' "$@"
 }
 
+printWords() {
```

Comment:
Err I meant to keep `printLines`, but I guess it's now unused so there's no point.